### ISSUE TITLE
indentation issue

### ISSUE THREAD

**Issue by mosheavni** [nvuillam/npm-groovy-lint#379]
I see some indentation issues
when calling a function that's defined like so:

```groovy
def myFunc(Map args=[:]) {
    println args.foo
    println args.bar
    println args.baz
}
```

and splitting the parameters in the function call like so:

desired output:

```groovy
myFunc(
    foo: 1,
    bar: 2,
    baz: 3,
)
```

actual output:

```groovy
myFunc(
foo: 1,
bar: 2,
baz: 3,
)
```

The parameters are not indented correctly.

Also, on a multiline string, it's not indented, maybe that is intentional to avoid adding unwanted indentation in multi line text blocks.

code:

```groovy
def myFunc() {
def multiLineText = '''
hello
line
another line
'''
}
```

desired output:

```groovy
def myFunc() {
    def multiLineText = '''
        hello
        line
        another line
    '''
}
```

actual:

```groovy
def myFunc() {
    def multiLineText = '''
hello
line
another line
'''
}
```

Maybe there's a rule to indent multiline text blocks?

**Comment by nvuillam** [comment]
@mosheavni this is probably a CodeNarc issue, and [Indentation rule](https://codenarc.org/codenarc-rules-formatting.html#Indentation-rule) is quite tricky and have some limitations...

> Known Limitations include:
>
> - Checks spaces only (not tabs)
> - Does not check comments
> - Does not check line-continuations (i.e., checks only the first line of a statement)
> - Does not check multiple statements/members on the same line (only checks the first one)
> - Does not check Map entry expressions
> - Does not check List expressions
> - Does not check calls to this() and super() within a constructor
> When classes, methods or fields have annotations, the indentation of the annotation is checked, not the actual member. And only the first annotation is checked, if there is more than one.

You can report it on their repository :)

https://github.com/CodeNarc/CodeNarc

**Comment by chrismair** [comment]
For what it's worth, it looks like CodeNarc is treating the expected indentation style as valid .. I showed the tests I implemented on https://github.com/CodeNarc/CodeNarc/issues/787.
